magento/pwa-studio
# Add to Wishlist from Cart

## Description

This PR adds a new feature that lets the user move an item from the cart to the wishlist. This PR only targets single wishlist use cases, for instance, CE backends or EE backends with multiple wishlists turned off.

## Related Issue
Closes PWA-1495

### Verification Stakeholders
@schensley 
@tkacheva 
@dpatil-magento 

### Verification Steps
1. Go to the deployed version of the app.
2. Login and add an item to the cart.
3. Go to the cart page and click on the kabab menu.
4. Click the `Save for later` button.
5. The item should be removed from the cart and should be available in the favorites list.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
